pom.xml

```diff
@@ -2611,9 +2611,9 @@
             </args>
             <jvmArgs>
               <jvmArg>-Xss128m</jvmArg>
-              <jvmArg>-Xms4g</jvmArg>
-              <jvmArg>-Xmx4g</jvmArg>
-              <jvmArg>-XX:MaxMetaspaceSize=2g</jvmArg>
+              <jvmArg>-Xms6g</jvmArg>
+              <jvmArg>-Xmx6g</jvmArg>
```

Comment:
Hi, @viirya . Currently, GitHub Action kills our job because it consume more memory. The main return code is 137 .  I'm not sure about this increament.